packages/core/src/components/graphs/scatter-stacked.ts

```diff
@@ -121,6 +121,17 @@ export class StackedScatter extends Scatter {
 				}
 			});
 		});
-		return tooltipData;
+
+		return this.model.getDisplayData().filter((d) => {
+			return (
+				tooltipData.find((ttd) => {
+					return (
+						ttd[groupMapsTo] == d[groupMapsTo] &&
+						ttd[domainIdentifier] == d[domainIdentifier] &&
+						ttd[rangeIdentifier] == d[rangeIdentifier]
```

Comment:
and updating the same names here. I think this'll make it a bit more readable

Reply:
I've pushed the changes with the updated names